pytorch_lightning/accelerators/cpu.py

```diff
@@ -13,8 +13,9 @@
 # limitations under the License.
 from __future__ import annotations
 
-from typing import Any
+from typing import Any, Dict
 
+import psutil
```

Comment:
`psutil` is an external dependency right? does this need to be guarded by an import like `if _PSUTIL_AVAILABLE` ? https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/imports.py